liberapay/exceptions.py

```diff
@@ -163,8 +163,8 @@ class TooManyLoginEmails(LazyResponseXXX):
     code = 429
     def msg(self, _):
         return _(
-            "You have consumed your quota of email logins, please try again tomorrow, "
-            "or contact support@liberapay.com."
+            "You have exceeded your quota of email logins, please try again tomorrow, "
```

Comment:
I would argue exceeded makes it more clear that this is a threshold, and that the user has passed it.